iree/compiler/Api/test/compile-mhlo-test-main.c

```diff
@@ -12,8 +12,8 @@
 // be the first user of the combined MLIR+IREE CAPI:
 // https://github.com/google/iree/pull/8582
 
-#include <iree-compiler-c/Compiler.h>
 #include <iree/base/string_builder.h>
+#include <iree/compiler/Api/Compiler.h>
```

Comment:
Everytime I see the capitalization I pause: "Api" just looks weird (api or API would be less jarring and more consistent with CAPI). Is what it signifies here that this is the directory for the public interface? I'm more used to having the implementation parts in its own internal dir.

Reply:
Ben already called that out - Most recent snapshot renames to API.